src/main/java/de/hysky/skyblocker/utils/scheduler/MessageScheduler.java

```diff
@@ -46,9 +46,11 @@ private void sendMessage(String message, boolean hide) {
             return;
         }
         message = StringHelper.truncateChat(StringUtils.normalizeSpace(message.trim()));
+        boolean isCommand = message.startsWith("/");
 
-		if (!hide) client.inGameHud.getChatHud().addToMessageHistory(message);
-        if (message.startsWith("/")) {
+		if (!hide && !isCommand) client.inGameHud.getChatHud().addToMessageHistory(message);
+
+        if (isCommand) {
```

Comment:
Sorry but I disagree with this, you should just set hide to true at the call site.

Reply:
There is way too many call sites to keep track of and the command history shouldn't be filled with commands the player themselves never sent especially since it keeps track of the last 50 and is restored for each session.

Reply:
I can do that and also remove the default override so devs will have to choose whether to add to history or not every time they use the method.